.github/workflows/build-frontend.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           cd frontend
           yarn pretty-quick --check --verbose --branch ${{ inputs.target_branch }}
           yarn test
-
+          
```

Comment:
??

Reply:
I'm removing these spaces on the next commit.